RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 Unreleased
 ---
 * [**] Prevent error message from unneccesarily firing when uploading to Gallery block [https://github.com/WordPress/gutenberg/pull/46175]
```

Comment:
Not related to this PR but I noticed that this entry is already in `1.85.1` section, so we might take the opportunity to remove it.